packages/dx-react-scheduler/src/plugins/all-day-panel.test.tsx

```diff
@@ -160,5 +161,23 @@ describe('AllDayPanel', () => {
       expect(tree.find(defaultProps.cellComponent).exists())
         .toBeTruthy();
     });
+
+    it('should render body template', () => {
+      const tree = mount((
+        <PluginHost>
+          {pluginDepsToComponents(defaultDeps)}
+          <AllDayPanel
+            {...defaultProps}
+          />
+        </PluginHost>
+      ));
+
+      const templatePlaceholder = tree
+        .find(Template)
+        .filterWhere(node => node.props().name === 'body');
```

Comment:
Can we use `findWhere()`?
https://airbnb.io/enzyme/docs/api/ReactWrapper/findWhere.html